pkgs/tools/inputmethods/fcitx5/fcitx5-sayura.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  extra-cmake-modules,
+  gettext,
+  fcitx5,
+}:
```

Comment:
This format is not align to the convention of Nixpkgs.  However, I will not request you to change this since we do not state that in the manual and do not have a formatter yet (see RFC 101).